### DISCUSSION TITLE
website wont open

### DISCUSSION THREAD

**Discussion by brjohnson0916** [community/community#34158] [Other Feature Feedback, Questions, & Ideas]
Hello, I am trying to open the GitHub website and it takes me to the page where it says "Hmm...took to long to respond." Can anyone help me with what to do?

**Comment by radar-blip** [comment]
You mean the main page? GitHub.com?
That was down a couple days ago.

**Comment by brjohnson0916** [comment]
Oh yes!! I can’t believe someone responded!

Is it back up? Why is it working for my friend and not me? I have tried
every browser and on my phone. Everything else works for me on my laptop
and phone

**Reply by radar-blip** [reply]
Some people had luck logging into their profile. `https://github.com/username`

**Comment by brjohnson0916** [comment]
I tried that and it didn’t work. Any other suggestions?

**Reply by radar-blip** [reply]
No sorry, just keep trying is all I can tell you.
